CRM/Utils/System.php

```diff
@@ -1488,8 +1488,7 @@ public static function flushCache() {
       = CRM_Contribute_BAO_Contribution::$_importableFields
         = CRM_Contribute_BAO_Contribution::$_exportableFields
           = CRM_Pledge_BAO_Pledge::$_exportableFields
-            = CRM_Core_BAO_CustomField::$_importFields
-              = CRM_Core_BAO_Cache::$_cache = CRM_Core_DAO::$_dbColumnValueCache = NULL;
+            = CRM_Core_BAO_Cache::$_cache = CRM_Core_DAO::$_dbColumnValueCache = NULL;
```

Comment:
@seamuslee001 `CRM_Core_BAO_Cache` should be replaced by `\Civi::cache()` I think - but could be a follow up PR.

Reply:
would rather deal with that separately to this

Reply:
:+1: